### DISCUSSION TITLE
災害ドメイン知識を貯めるdiscussion

### DISCUSSION THREAD

**Discussion by suguru-toyohara** [tech-creative-club/disaster-posts#16] [General]
# 災害に必要な情報はなんだろう

災害時に必要な情報、いろいろなパターン分けができると思います。
それについてドメイン知識を深めるためのdiscussion

**Comment by suguru-toyohara** [comment]
### 個では意味がないが群で意味が生まれる情報

- 「停電」「断水」「孤立」「物資不足」「避難中」といった位置情報＋情報は、個では意味を為さずとも、群で意味を形成できる。
- これは真偽不明情報に対して強力な対抗策になりうる。

**Reply by suguru-toyohara** [reply]
そもそも必要なのって群としての統計情報であり、それ以外は基本的にはノイズであり必要ないのでは？という気持ちになる。

**Comment by Ryokugyoku** [comment]
群で意味を形成する情報だと他にも
「被害状況」などが挙げられそうです。
例えば　〇〇の道路が封鎖して通れない、〇〇で火災が発生していて危険といった情報を収集することが大切な気がします。
他に逃げる時に欲しくなる情報だと
食料を提供している地位域などでしょうか？
